### DISCUSSION TITLE
Why does terser-webpack-plugin exist in dependencies but not in devDependencies?

### DISCUSSION THREAD

**Discussion by yoghurtxu** [webpack/webpack#18106] [Q&A]
Why does terser-webpack-plugin exist in dependencies but not in devDependencies?
https://www.npmjs.com/package/terser-webpack-plugin:
The documentation of terser-webpack-plugin is recommended to be written in devDependencies.

**Comment by alexander-akait** [comment]
Because it is dependency for webpack, but it is devDependencies for your application
